Makefile

```diff
@@ -94,7 +94,12 @@ endif
 #####
 
 CARGO_BINARY ?= cargo
-CARGO_TARGET ?=
+CARGO_TARGET ?= 
+CARGO_TARGET_STR ?=
+
+ifneq ($(CARGO_TARGET),)
+CARGO_TARGET_STR := --target $(CARGO_TARGET)
```

Comment:
This should be named `CARGO_TARGET_FLAG`: it's way clearer

Reply:
Done in https://github.com/wasmerio/wasmer/pull/3430/commits/5e096a0af50b2270559dbc266f86e05eb388f1b3